views/pages/DashboardPage/DashboardPage.test.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react'
+import { render, screen, within, waitFor } from '@testing-library/react'
+import { AuthContext, NotificationContext } from '../../contexts'
+import { MemoryRouter } from 'react-router-dom'
+import DashboardPage from '.'
+
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useOutletContext: () => {
+    return {
+      user: {
+        display_name: 'Owl',
+        icon: 'data:image/png;base64,SGVsbG8sIFdvcmxkIQ==',
+        iconFileName: 'profileImage.png',
+        uid: '1234'
+      }
+    }
+  }
+}))
+jest.mock('../../api', () => ({
+  participants: {
+    all: () => Promise.resolve([
+            {
```

Comment:
I think it's worth making this a fixture along with the others in `test/fixtures.js`. That way if/when other attributes are added we only need them added in 1 place.

Reply:
Totally agree, thanks!

Reply:
3433ed5